packages/block-library/src/missing/edit.native.js

```diff
@@ -18,26 +18,86 @@ import { __ } from '@wordpress/i18n';
  */
 import styles from './style.scss';
 
+const isAndroid = Platform.OS === 'android';
+const platformText = isAndroid ? 'Android' : 'iOS';
 export class UnsupportedBlockEdit extends Component {
+	constructor( props ) {
+		super( props );
+		this.state = { showHelp: false };
+	}
+
+	toggleSheet() {
+		this.setState( {
+			showHelp: ! this.state.showHelp,
+		} );
+	}
+
+	renderHelpIcon() {
+		return <TouchableWithoutFeedback
+			accessibilityLabel={ __( 'Help icon' ) }
+			accessibilityRole={ 'button' }
+			accessibilityHint={ __( 'Tap here to show help' ) }
+			onPress={ this.toggleSheet.bind( this ) }
+		>
+			<View style={ styles.helpIconContainer } >
+				<Icon
+					className="unsupported-icon-help"
+					label={ __( 'Help icon' ) }
+					icon="editor-help"
+				/>
+			</View>
+		</TouchableWithoutFeedback>;
+	}
+
+	renderSheetButton( text, action ) {
+		return <TouchableOpacity
+			onPress={ action() }
+			style={ styles.sheetButton }
+		>
+			<Text style={ styles.sheetButtonText } >{ text }</Text>
+		</TouchableOpacity>;
+	}
+
+	renderSheet( title ) {
+		return <BottomSheet
+			isVisible={ this.state.showHelp }
+			hideHeader
+			onClose={ this.toggleSheet.bind( this ) }
+		>
+			<View style={ styles.infoContainer } >
+				<Icon icon="editor-help" style={ styles.infoIcon } size={ styles.infoIcon.size } />
+				<Text style={ [ styles.infoText, styles.infoTitle ] }>
+					{ __( '\'' + title + '\' isn\'t yet supported on WordPress for ' + platformText ) }
```

Comment:
You can't compose strings inside `__()`, you need to use `sprintf` instead and add a placeholder for the title. For the platform, I think it's safer to have one string for iOS and one for Android, since it's more obvious for translators.

Reply:
Addressed with 4c2b904.